docs/source/software/quick_start_guide.rst

```diff
@@ -108,5 +108,33 @@ QGroundControl and change the following:
 * EKF2_HGT_MODE = vision
 
 
-Safety  
-=======================
\ No newline at end of file
+Safety Checklist
+==================
+
+Make sure you complete the following check list *before every flight*. Most of the checks are automated by the controller on startup, but that is no substitute for good ole human caution.
+
+
+Hardware
+----------
+
+1. Mechanical
+    a. All *4* rotors are tightly secured
+    b. Battery is secured 
+2. Electrical 
+    a. The fcu serial connection is secured between the px4 and cc
+    b. Battery voltage is over the nominal 11.5v (*DO NOT* start a flight with less than this voltage)
+
+
+Software
+--------
+
+1. Motive
+    a. z is up
+    b. quad frame is aligned with room (room's long-axis is quad x-axis)
+2. ROS Topics
+    a. ``local_position/pose`` matches ``vision_pose/pose``
+    b.  ``state`` has ``connected=True``
+3. Quad arms on the ground 
+    a. This can be checked without even ssh-ing the quad
+    b. You will not have attitude control
+    c. Do not try to takeoff, just arm check propeller rotation and disarm
```

Comment:
I think the above Software 1. and 2. look good, but I'm not sure I specifically follow what you mean by 3. I think some elaboration on each of these points might be useful.

I think this checklist is great for running through to verify safety before flying. However, for all the points in this checklist, would it be beneficial to have an elaborated page of the safety guide? There, maybe pictures or elaboration could be useful for someone who has never used the system before. This could include the comment I made earlier about the meaning of "secured".

Reply:
done